openrank-sdk/src/main.rs

```diff
@@ -1,74 +1,10 @@
-use alloy_rlp::encode;
 use clap::{Parser, Subcommand};
-use csv::StringRecord;
 use dotenv::dotenv;
-use getset::Getters;
-use jsonrpsee::{core::client::ClientT, http_client::HttpClient};
-use k256::{ecdsa::Error as EcdsaError, ecdsa::SigningKey, schnorr::CryptoRngCore};
-use openrank_common::{
-    address_from_sk,
-    algos::et::is_converged_org,
-    merkle::hash_leaf,
-    query::{GetSeedUpdateQuery, GetTrustUpdateQuery},
-    topics::Domain,
-    tx::{
-        self,
-        compute::{self, Verification},
-        consts,
-        trust::{ScoreEntry, SeedUpdate, TrustEntry, TrustUpdate},
-        Address, Body, Tx, TxHash,
-    },
-    tx_event::TxEvent,
-};
-use rand::{thread_rng, Rng};
-use serde::{Deserialize, Serialize};
-use sha3::Keccak256;
-use std::{
-    cmp::Ordering,
-    collections::HashMap,
-    fs::File,
-    io::{Read, Write},
-    process::ExitCode,
-};
-use std::{
-    io::{self, BufWriter},
-    num,
-};
-
-const TRUST_CHUNK_SIZE: usize = 500;
-const SEED_CHUNK_SIZE: usize = 1000;
-const NOT_FOUND_CODE: i32 = -32016;
-const TRANSIENT_STATUS_CODE: u8 = 2;
```

Comment:
`TRANSIENT_STATUS_CODE` should be in `main.rs` since it is returned by the CLI

Reply:
Done https://github.com/openrankprotocol/openrank/pull/209/commits/dbe63417e53e75927f600b877412060bd970137b